libr/main/rabin2.c

```diff
@@ -1164,6 +1209,9 @@ R_API int r_main_rabin2(int argc, char **argv) {
 				"plugins found!\n", bin->file);
 		}
 	}
+	if (sss == 3) {
```

Comment:
uhm, -SSS right now is the same as passing -S -SS, not sure if that will be what we want :?

Reply:
Actually... this is true anyway. And also `-SS` is equal to `-S -S` right now.

Reply:
Yeah, and -SSS is already equal to -S -SS even without this.

Reply:
Thanks for the code! 🍻